src/shared/globalStyles.js

```diff
@@ -47,13 +53,48 @@ export const GlobalStyles = createGlobalStyle`
   h3,
   h4,
   h5,
-  h6 {
+  h6 {   
+    font-family: ${(props) => props.theme.global.fonts.serif};
     color: ${(props) => props.theme.global.colors.black};
-    font-family: 'Merriweather', serif;
     font-weight: bold;
-    line-height: 1.3;
+    line-height: 1.2;
+    margin : 0;
```

Comment:
```suggestion
    margin : 0 0 24px;
```

Use this global style for heading margins instead of styled components. Adjust whatever needs to be adjusted